modules/simple-payments/paypal-express-checkout.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * This PaypalExpressCheckout global is included by wp_enqueue_script( 'paypal-express-checkout' );
+ * It handles communication with Paypal Express checkout and public-api.wordpress.com for the purposes
+ * of simple-payments module.
+ */
+
+/* global paypal */
+/* exported PaypalExpressCheckout */
+/* jshint unused:false */
+var PaypalExpressCheckout = {
+	constants: {
+		createPaymentEndpoint: '', //TODO: point to the actual endpoint
+		executePaymentEndpoint: '' //TODO: point to the actual endpoint
+	},
+	renderButton: function( id ) {
+		paypal.Button.render( {
+			commit: true,
+			style: {
+				label: 'pay',
+				color: 'blue'
+			},
+			payment: function() {
+				return paypal.request.post( PaypalExpressCheckout.constants.createPaymentEndpoint ).then( function( data ) {
```

Comment:
`paypal.request.post` accepts post data as the second argument: https://github.com/paypal/paypal-checkout/blob/master/docs/http.md#windowpaypalrequestposturl-data-options

I know we don't have the wpcom REST endpoints ready but might be a good time to add what we want to send to the `createPaymentEndpoint`. 

I'm not sure what we can derive from the request but I'd imagine we'd want to send something like:
```
{
    button_id: 123,
    post_id: 456,
    is_jetpack: true,
    button_metadata: { ... } // for quantity etc
}
```